bin/sync_django_ldap.py

```diff
@@ -39,6 +39,7 @@
 NAGIOS_HEADER = "sync_django_to_ldap"
 NAGIOS_CHECK_INTERVAL_THRESHOLD = 15 * 60  # 15 minutes
 SYNC_TIMESTAMP_FILENAME = "/var/cache/%s.timestamp" % (NAGIOS_HEADER)
+CHILD_USERS = ['ldap', 'apache']
```

Comment:
make this an option to pass in the commandline?

Reply:
good idea, this can now be overwritten with `--user`